code4lib/2022.code4lib.org
# Community support

**Ready for review**. To test:

- visit /conduct/index.html
- there should be a list of onsite & online volunteers on the page
- inspect images to ensure their alt text is present for all (some will be only a name, some will be full description, we couldn't get descriptions for everyone)
- there should be complete schedules for onsite and online volunteers
  + Eric will ask the CSS Coordinator to double-check these once they are live
